app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientEvent.kt

```diff
@@ -1,15 +1,9 @@
 package org.simple.clinic.summary.linkId
 
-import org.simple.clinic.patient.businessid.Identifier
 import org.simple.clinic.widgets.UiEvent
-import java.util.UUID
 
 sealed class LinkIdWithPatientEvent : UiEvent
 
-data class LinkIdWithPatientViewShown(val patientUuid: UUID, val identifier: Identifier) : LinkIdWithPatientEvent() {
-  override val analyticsName: String = "LinkIdWithPatient:Sheet Created"
```

Comment:
If I am not mistaken, we might be using this event in some Mixpanel reports to see how the `Link Id With Patient` sheet is used. We might have to update those reports to also take the `ScreenChange` event into account as well.

Reply:
I have overriden the analytics name for the key, so we should still know when the sheet is opened. But the analytics name is not the same as this, what should I be doing here?

Analytics name is `Link ID With Patient Sheet`

Reply:
Earlier, this would have been reported as a `UserInteraction` event. However, now it would be a `ScreenChange` event with the `incoming` set to `Link ID With Patient Sheet`.

We should be looking for reports which deal with the link sheet and update these, basically.